### ISSUE TITLE
Can you Pre-Select a tab?

### ISSUE THREAD

**Issue by kaddyuk** [RibbonWinForms/RibbonWinForms#62]
I have tabs 1 through 4, when the form opens however, I want to have tab 3 selected (tab 1 is selected by default).

Can I do this programatically?

Thanks

**Comment by krifalk** [comment]
Yes, that's quite simple. Just set the desired tab with Ribbon.ActiveTab:

`ribbon1.ActiveTab = tab3;`
